.github/workflows/backport_branches.yml

```diff
@@ -3,6 +3,9 @@ name: BackportPR
 env:
   # Force the stdout and stderr streams to be unbuffered
   PYTHONUNBUFFERED: 1
+  # Export system tables to ClickHouse Cloud
+  CLICKHOUSE_CI_LOGS_HOST: ${{ secrets.CLICKHOUSE_CI_LOGS_HOST }}
```

Comment:
We usually use the SSM for this purpose, and this function https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/get_robot_token.py#L18